projects/js-packages/charts/src/providers/chart-context/themes.ts

```diff
@@ -151,6 +151,15 @@ const wooTheme: ChartTheme = {
 	legendContainerStyles: {
 		gap: '8px',
 	},
+	legendShapeStyles: [
```

Comment:
Copied from Woo Analytics. We should be able to remove it there after this is shipped